### ISSUE TITLE
ML filter error

### ISSUE THREAD

**Issue by mihshimada** [ConesaLab/SQANTI3#169]
Hello,

I'm trying to run the ML filter, and now I get the following error on both test data and my own data, even if I don't have any specific options.

*******
Error in Random forest evaluation: applying classifier to test set...
 predict.randomForest(modelFit, newdata, type = "prob") 
variables in the training data missing in newdata
predict ... probFunction -> <Anonymous> -> predict -> predict.randomForest
*******

Please let me know what I should do.

Thank you very much.
